core/src/main/java/com/linecorp/armeria/internal/server/ResponseConversionUtil.java

```diff
@@ -248,7 +252,9 @@ public void onComplete() {
                 return;
             }
             if (!trailers.isEmpty()) {
-                writer.write(trailers);
+                if (!writer.tryWrite(trailers)) {
+                    logger.warn("Failed to write a trailers: {}", trailers);
```

Comment:
Probably better not logging, because a response can be closed by a remote peer at any time?

Reply:
Removed that. Thanks!